index.js

```diff
@@ -8,6 +8,10 @@ var request = require('request');
 var cookieJar = request.jar();
 var sessionID = null;
 
+var statusInterval;
```

Comment:
these variables cannot be in global space because they'll be in conflict if people have multiple instances.

Reply:
@bb21b just making sure you saw this review. :)

Reply:
@krocheck @josephdadams Thanks for the ping, it'll be a couple of weeks until I get some free time to sort this. But it is good to be busy again! 👍

Reply:
@BB21B let me know when you circle back to this.  I would make the change myself but it would still need to be tested :/

Reply:
@krocheck Hey I'm going to get back on this. Just getting my brain back in the right gear :P

Reply:
@bb21b just guessing that my original work on this with the existing variables also need to be moved into the right scope as well. `instance.prototype.statusInterval;` etc.